OpenRA.Mods.Common/Activities/Move/MoveAdjacentTo.cs

```diff
@@ -161,7 +161,7 @@ public override IEnumerable<Target> GetTargets(Actor self)
 			return Target.None;
 		}
 
-		public override bool Cancel(Actor self)
+		public override bool Cancel(Actor self, bool keepQueue = false)
```

Comment:
Ah right, I was thinking that this would chain through the activity queue, but it doesn't.